neo/SmartContract/Helper.cs

```diff
@@ -13,6 +21,97 @@ public static class Helper
         private static readonly ConcurrentDictionary<string, uint> MethodHashes
             = new ConcurrentDictionary<string, uint>();
 
+        public static StackItem DeserializeStackItem(this byte[] data, uint maxArraySize)
+        {
+            using (MemoryStream ms = new MemoryStream(data, false))
+            using (BinaryReader reader = new BinaryReader(ms))
+            {
+                return DeserializeStackItem(reader, maxArraySize);
+            }
+        }
+
+        private static StackItem DeserializeStackItem(BinaryReader reader, uint maxArraySize)
+        {
+            Stack<StackItem> deserialized = new Stack<StackItem>();
+            int undeserialized = 1;
+            while (undeserialized-- > 0)
```

Comment:
I prefer to put: `while(underserialized > 0)` and decrement on the next line `underserialized--`.
This greatly improves safety and readability, in my opinion.

Reply:
There is no difference.

Reply:
> There is no difference

For smart guys only... the rest of us need to think about it :)